app/multitenant/dynamo_collector.go

```diff
@@ -71,22 +86,26 @@ type DynamoDBCollector interface {
 }
 
 type dynamoDBCollector struct {
-	userIDer  UserIDer
-	db        *dynamodb.DynamoDB
-	tableName string
-	merger    app.Merger
-	cache     gcache.Cache
+	userIDer   UserIDer
+	db         *dynamodb.DynamoDB
+	s3         *s3.S3
+	tableName  string
+	bucketName string
+	merger     app.Merger
+	cache      gcache.Cache
 }
 
 // NewDynamoDBCollector the reaper of souls
 // https://github.com/aws/aws-sdk-go/wiki/common-examples
-func NewDynamoDBCollector(config *aws.Config, userIDer UserIDer, tableName string) DynamoDBCollector {
+func NewDynamoDBCollector(dynamoDBConfig, s3Config *aws.Config, userIDer UserIDer, tableName, bucketName string) DynamoDBCollector {
```

Comment:
It's not really a `DynamoCollector` anymore, is it. It's more of an `AWSCollector`, but...